test/homepageTests.js

```diff
@@ -0,0 +1,55 @@
+const Browser = require('zombie');
+
+import server from '../server.js';
+const browser = new Browser();
+
+describe('Visiting the site loggged off', function(){
+
+	before(function(done){
+		browser.visit('http://localhost:3000/', done);
```

Comment:
Since this doesn't actually start a server, it assumes that the `calendarTest.js` is run first.  In general that is bad practice for a test.

Why not do something of the like in each file:

```javascript
var http;
before(() => {
  http = server.listen(...);
});

after(() => {
  http.close();
});